release-notes/opensearch-release-notes-1.0.0-rc1.md

```diff
@@ -0,0 +1,324 @@
+* Allow attempt to load security config in case of plugin restart even if security index already exists ([#1154](https://github.com/opensearch-project/security/pull/1154))
+* Allowing granular access for data-stream related transport actions ([#1170](https://github.com/opensearch-project/security/pull/1170))
+
+
+### Opensearch Security Dashboards Plugin
+* Add svg content [#766](https://github.com/opensearch-project/security-dashboards-plugin/pull/766)
+* Changes regarding the anonymous auth flow [#768](https://github.com/opensearch-project/security-dashboards-plugin/pull/768)
+
+
+## BUG FIXES
+
+### Opensearch Alerting Dashboards Plugin
+* Update .opensearch_dashboards-plugin-helpers.json ([#8](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/8))
+* Fix AD backend name to opensearch-anomaly-detection ([#9](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/9))
+
+
+### Opensearch Dashboards Notebooks
+* Fix run all paragraphs for visualizations ([#20](https://github.com/opensearch-project/dashboards-notebooks/pull/20))
+
+
+### Opensearch Dashboards Notebooks
+* Fix run all paragraphs for visualizations ([#20](https://github.com/opensearch-project/dashboards-notebooks/pull/20))
+
+
+### Opensearch Dashboards Reports
+* Configure index to have default one replica and upper bound 2 ([#62](https://github.com/opensearch-project/dashboards-reports/pull/62))
+* Fix case-sensitive directory name for chromium zip ([#35](https://github.com/opensearch-project/dashboards-reports/pull/35))
+* Add condition to fix negative value display ([#51](https://github.com/opensearch-project/dashboards-reports/pull/51))
+* Fix csv parsing function ([#53](https://github.com/opensearch-project/dashboards-reports/pull/53))
+
+
+### Opensearch Index Management Dashboards Plugin
+* Rename plugin helper file ([#11](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/11))
+
+
+### Opensearch Performance Analyzer
+* Create conf directory if not exist to avoid NoSuchFile exceptions ([#9](https://github.com/opensearch-project/performance-analyzer/pull/9))
+* Update docker artifact link ([#9](https://github.com/opensearch-project/performance-analyzer-rca/pull/9))
+* Modify JVM gen tuning policy and update the Unit test ([#12](https://github.com/opensearch-project/performance-analyzer-rca/pull/12))
+
+
+### Opensearch Security
+* Delay the security index initial bootstrap when the index is red ([#1153](https://github.com/opensearch-project/security/pull/1153))
+* Remove redundant isEmpty check and incorrect string equals operator ([#1181](https://github.com/opensearch-project/security/pull/1181))
+
+
+### Opensearch Security Dashboards Plugin
+* Fixing JSON parsing in SAML for strings that contain '' character [#749](https://github.com/opensearch-project/security-dashboards-plugin/pull/749)
+* Fix login redirect [#777](https://github.com/opensearch-project/security-dashboards-plugin/pull/777)
```

Comment:
PR info not enclosed in (), not following the same standard as others.

Reply:
Added